### DISCUSSION TITLE
Document file system structure of service parameters

### DISCUSSION THREAD

**Discussion by timirnich** [epinio/epinio#819] [Ideas]
Epinio mounts the service credentials secret generated by minibroker into the file system of the run container under /services/<servicename>/*. Each service seems to have it's own convention which parameters get created and how they are named. Currently, the user needs to find this out by creating the service and then `kubectl describe` on the secret. This should be documented for the currently supported service types, and, ideally, also output when creating the service.

**Comment by agracey** [comment]
Chatting with Tim right now about this. 

We should also look at listing the parameters exposed in each service through `epinio service show <service name>`.

We do this for custom service, why not marketplace services?

**Reply by manno** [reply]
We need to add documentation on services. (https://github.com/epinio/epinio/issues/820)
Story about service parameters (#821)
